File: controlePedidosFront/src/app/shared/models/cliente.ts
export class Cliente {
  id: number;
  tipo: string;
  documento: string;
  nome: string;

  constructor(codigo: number, tipo: string, documento: string, nome:string){
    this.id = codigo;
    this.tipo = tipo;
    this.documento = documento;
    this.nome = nome;
  }
}

File: Chocolate-ponto-e-virgula-feature-back_end/controlePedidos/src/main/java/com/br/chocolatePontoVirgula/model/entity/Cliente.java
package com.br.chocolatePontoVirgula.model.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "cliente")
public class Cliente {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "codigo")
	private Long id;

    @Column(name = "tipo")
	private String tipo;

    @Column(name = "documento")
	private String documento;

    @Column(name = "nome")
	private String nome;

	


}
